de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/hibernate/WikiHibernateUtil.java

```diff
@@ -109,10 +112,10 @@ else if(jdbcURL.toLowerCase().contains("hsql")) {
         p.setProperty("hibernate.show_sql","false");
 
         // Do only update schema on changes
-        if(useMySQL) {
+        if(useMySQL || useMariaDB) {
```

Comment:
Having useMariaDB in both conditions seems pointless.

Reply:
Looked in the history. Looks like a refactoring error and should be hsqldb. I will fix it. Nice catch!